x-pack/solutions/observability/plugins/apm/public/components/app/transaction_details/waterfall_with_summary/waterfall_container/waterfall/waterfall_helpers/waterfall_helpers.ts

```diff
@@ -22,6 +22,7 @@ import type {
 type TraceAPIResponse = APIReturnType<'GET /internal/apm/traces/{traceId}'>;
 
 const ROOT_ID = 'root';
+const HIDDEN_ID = 'hidden';
```

Comment:
This is confusing. Could we rename it `ORPHAN_ID = 'orphan_span'` or sth like that? If I understood correctly, `parent.id` assigned with `HIDDEN_ID` won't actually be hidden, will they?